.projenrc.js

```diff
@@ -7,6 +7,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
   authorOrganization: true,
   cdkVersion: '2.8.0',
   defaultReleaseBranch: 'main',
+  description: 'The CDK Construct Library that helps you build ECS services using simple extensions',
```

Comment:
feel free to modify this however you see fit, I copied it from what it was in v1.